inc/sccm.class.php

```
@@ -49,6 +49,26 @@ public function showHome()
         echo __('Please, read the documentation before using that.', 'footprints');
     }
 
+    public function getSccmBuildNumber()
+    {
+        $PluginSccmSccmdb = new PluginSccmSccmdb();
+        $res = $PluginSccmSccmdb->connect();
+        if (!$res) {
+            die;
```

To avoid `Die`, maybe you can throw an exception?